src/Microsoft.TemplateEngine.Cli/CommandParsing/CommandLineHelpViewer.cs

```diff
@@ -0,0 +1,120 @@
+using Microsoft.DotNet.Cli.CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    internal static class CommandLineHelpViewer
```

Comment:
minor: if this is an extension class, consider the suffix "Extension" for the class name